VS 21 February/VS 21 February/Program.cs

```diff
@@ -2,8 +2,186 @@
 {
     static void Main(string[] args)
     {
-        string name = Console.ReadLine();
+        Lesson2_Task4_Way2();
+
+    }
+
+
+    public static void Lesson1_Task1()
+    {
+        char name = char.Parse(Console.ReadLine());
         Console.WriteLine("Hello, World!");
         Console.WriteLine("Hello, " + name);
     }
+
+
+    public static void Lesson2_Task1()
+    {
+        int operand1 = 25;
+        int operand2 = 5;
+
+        string sign = Console.ReadLine();
+        int result;
+
+        switch(sign)
+        {
+            case "+":
+                result = operand1 + operand2;
+                Console.WriteLine($"addition operation: {operand1} {sign} {operand2} = {result}");
+                break;
+
+            case "-":
+                result = operand1 - operand2;
+                Console.WriteLine($"subtraction operation: {operand1} {sign} {operand2} = {result}");
+                break;
+
+            case "*":
+                result = operand1 * operand2;
+                Console.WriteLine($"multiplication operation: {operand1} {sign} {operand2} = {result}");
+                break;
+
+            case "/":
+                if ((operand2 != 0))
+                {
+                    result = operand1 / operand2;
+                    Console.WriteLine($"division operation: {operand1} {sign} {operand2} = {result}");
+                }
+                else
+                {
+                    Console.WriteLine($"Error! Dividing by Zero impossible");
+                }   
+                break;
+
+            default:
+                Console.WriteLine($"The calculator can only perform operations of addition, subtraction, division, multiplication.\nPlease enter a sign only for available operations.");
+                break;
+
+        } 
+    }
+
+
+    public static void Lesson2_Task2()
+    {
+        int number = int.Parse(Console.ReadLine());
+
+        if ((number >= 0) && (number <= 14))
+        {
+            Console.WriteLine($"The entered number is between 0 and 14");
+        }
+        else if ((number >= 15) && (number <= 35))
+        {
+            Console.WriteLine($"The entered number is between 15 and 35");
+        }
+        else if ((number >= 36) && (number <= 50))
+        {
+            Console.WriteLine($"The entered number is between 36 and 50");
+        }
+        else if ((number >= 51) && (number <= 100))
+        {
+            Console.WriteLine($"The entered number is between 51 and 100");
+        }
+        else
+        {
+            Console.WriteLine($"The entered number is not included in any of the existing ranges. Please enter integer number from 0 to 100");
+        }
+    }
+
+
```

Comment:
extra lines

Reply:
one empty line before and after methods

Reply:
Performed all required changes. Please, check.